### PR TITLE
Update Client Zone

### PR DESCRIPTION
**Pull request by Devan74**

### CODE REVIEW

File: src/components/Client/AddPayment.jsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            </div>
+            <div className="mb-4">
+              <label className="block text-gray-700 text-sm font-bold mb-2" for="status">
+                Status
+              </label>
+              <select
+                className="shadow border rounded w-full py-2 px-3 text-gray-700  focus:outline-none focus:shadow-outline"
+                id="status"
+              >
+                <option value="paid">Choose a status</option>
```
Comment (sathishkumar04vsk) [inline src/components/Client/AddPayment.jsx:65]: make the options in as object

File: src/components/Client/AddPayment.jsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                <div className="w-3/4">
+                  <label className="block text-gray-700 text-sm font-bold mb-2" for="bank">
+                    Select Bank
+                  </label>
+                  <select
+                    className="shadow border rounded w-full py-2 px-3 text-gray-700  focus:outline-none focus:shadow-outline"
+                    id="bank"
+                  >
+                    <option value="paid">Choose a Bank</option>
+                    <option value="paid">IOB</option>
```
Comment (sathishkumar04vsk) [inline src/components/Client/AddPayment.jsx:111]: @Devan74  options to object

File: src/components/Client/InvoiceTemplate.jsx [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+import BizzPassImage from "../../assets/images/bizzpass.png"
+import QrCodeImg from "../../assets/images/Qr code 2.png"
+import { MdOutlineCloudDownload } from "react-icons/md"
+import NavHeader from "../NavHeader"
+import RowNavigation from "./RowNavigation"
+
+function InvoiceTemplate() {
+  const { status } = useParams()
+  const error = false
+  const isLoading = false
```
Comment (Akash-pugazh) [inline src/components/Client/InvoiceTemplate.jsx:18]: for wat purppose

File: src/components/Client/AddPayment.jsx [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+                  cols="20"
+                  rows="4"
+                ></textarea>
+              </div>
+            </form>
+          </div>
+          <div className="w-[560px] h-[420px] border rounded-lg bg-gray-100 p-5">
+            <div className="">
+              <ul>
+                <li className="flex justify-between py-3">
```
Comment (sathishkumar04vsk) [inline src/components/Client/AddPayment.jsx:151]: don't repeate the element again and make the values form object

**Pull request merged**